src/components/main/Main.js

```diff
@@ -1,33 +1,26 @@
-import { v4 as uuidv4 } from 'uuid';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
 import Typography from '@mui/material/Typography';
 import Container from '@mui/material/Container';
 import { Box, Grid } from '@mui/material';
 import CardItem from '../card/card-item/CardItem';
 import CardDialog from '../card/card-dialog/CardDialog';
 
-const CARDS = [
-  {
-    id: uuidv4(),
-    title: 'Card 1',
-    body: 'Text of Card 1',
-  },
-  {
-    id: uuidv4(),
-    title: 'Card 2',
-    body: 'Text of Card 2',
-  },
-  {
-    id: uuidv4(),
-    title: 'Card 3',
-    body: 'Text of Card 3',
-  },
-];
-
 function Main() {
-  const [cards, setCards] = useState(CARDS);
+  const [cards, setCards] = useState([]);
+
+  useEffect(() => {
+    axios.get('https://jsonplaceholder.typicode.com/posts')
+      .then((response) => {
+        setCards(response.data);
+      });
+  }, []);
```

Comment:
Should I re-request when cards is changes? 
Add new card -> card variable changed -> post request -> get request (useEffect)

Reply:
No, we need that only for initialization